lib/metanorma/cli/commands/config.rb

```diff
@@ -0,0 +1,110 @@
+require "pathname"
+
+module Metanorma
+  module Cli
+    module Commands
+      class Config < Thor
+        class_option :global, aliases: "-g", type: :boolean, default: false, desc: "Use global config"
+
+        desc "get NAME", "Get config value"
+        def get(name = nil)
+          config_path = Metanorma::Cli.config_path(options.global)
+          write_default_config(config_path) unless File.exists?(config_path)
+
+          if name.nil?
+            print File.read(config_path, encoding: "utf-8")
+          else
+            config = ::YAML::load_file(config_path) || {}
+            ypath = name.split(".").map(&:to_sym)
+
+            print(config.dig(*ypath) || "nil")
+          end
+        end
+
+        desc "set NAME VALUE", "Set config value"
+        def set(name, value = nil)
+          config_path = Metanorma::Cli.config_path(options.global)
+          write_default_config(config_path) unless File.exists?(config_path)
+
+          config = ::YAML::load_file(config_path) || {}
+
+          value = case value
+                  when "true"
+                    true
+                  when "false"
+                    false
+                  else
+                    value
+                  end
+
+          ypath = name.split(".").map(&:to_sym)
+          deep_set(config, value, *ypath)
+
+          p config
+
```

Comment:
Use strings in yaml instead of symbols:

```yaml
---
:templates:
- :name: iso
  :source: https://github.com/metanorma/mn-templates-iso
  :type: custom
```
=> 

```yaml
---
templates:
- name: iso
  source: https://github.com/metanorma/mn-templates-iso
  type: custom
```

Reply:
Done